challenge3/webapp/msaito/app/controllers/statics_controller.rb

```diff
@@ -0,0 +1,31 @@
+class StaticsController < ApplicationController
+  before_action :search
+  helper_method :sort_column, :sort_direction 
+
+  def index
+    @houses = House.search(params).page(params[:page]).per(10).order(sort_column + " " + sort_direction)
```

Comment:
```suggestion
    @houses = House.search(params).page(params[:page]).per(10).order(sort_column => sort_direction)
```
と書けた気がする。
まぁ、好みの問題レベルなので、このままで問題ないです。